src/main/java/lotto/domain/Hit.java

```diff
@@ -0,0 +1,57 @@
+package lotto.domain;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Objects;
+
+public class Hit {
+    private static final int ZERO = 0;
+    private static final int MINIMUN_MATCH_COUNT = 3;
+    private final Map<Statistics, Integer> hit;

```

Comment:
```suggestion
    private static final int ZERO = 0;
    private static final int MINIMUN_MATCH_COUNT = 3;
    
    private final Map<Statistics, Integer> hit;

```
상수와 인스턴스 변수는 빈라인을 통해 구분해주면 가독성이 더 좋을 것 같아요! 😄